.github/workflows/require-build.yml

```diff
@@ -0,0 +1,25 @@
+name: Require built CSS
```

Comment:
Require makes me think this is importing something. I wonder if "ensure prebuilt CSS" would be clearer in the future?

Reply:
Yeah, I can see that connotation for "require" - I'll make it "Ensure pre-built CSS". The filename will change as well.